app/assets/stylesheets/resources/donations.scss

```diff
@@ -0,0 +1,3 @@
+div#donation-comment {
+  padding: 0px 20px;
+}
```

Comment:
I think you can utilize the bootstrap's class px-1 instead of defining this? Does adding `class='px-1'` remove the need to add a new stylesheet? If so, I'd suggest that.

Reply:
Unfortunately that doesn't line up the text: 
<img width="1438" alt="Screen Shot 2021-04-01 at 8 12 49 AM" src="https://user-images.githubusercontent.com/65036872/113307267-31198980-92c2-11eb-8e5f-1d0303d6dd2e.png">

I could just reuse the partner-notes div but I thought maybe that would look weird in the code:
<img width="582" alt="Screen Shot 2021-04-01 at 8 16 41 AM" src="https://user-images.githubusercontent.com/65036872/113307637-98373e00-92c2-11eb-8eab-8ce095952f72.png">

Reply:
@PhilipDeFraties ahh okay. That's a bummer :(. In that case, I think this is suitable for our objectives.

I forsee a redesign in the future so we can worry about it then.